test/basic/acm-dns-validated-create/acm_test.go

```diff
@@ -0,0 +1,92 @@
+package acm
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	legacy_aws_sdk_acm "github.com/aws/aws-sdk-go/service/acm"
+	terratest_aws "github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+
+	cac "github.com/aptible/cloud-api-clients/clients/go"
+	"github.com/aptible/terraform-provider-aptible-iaas/internal/client"
+)
+
+func cleanupAndAssert(t *testing.T, terraformOptions *terraform.Options) {
+	terraform.Destroy(t, terraformOptions)
+	// test / assert all failures here
+}
+func TestACMDnsValidated(t *testing.T) {
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: ".",
+		Vars: map[string]interface{}{
+			"organization_id": os.Getenv("ORGANIZATION_ID"),
+			"environment_id":  os.Getenv("ENVIRONMENT_ID"),
+			"aptible_host":    os.Getenv("APTIBLE_HOST"),
+			"dns_account_id":  "821755571104",
```

Comment:
can this use an env var? so we can test it somewhere and raise an error if it's missing in an init() func in this file?